cmd/draft/delete.go

```diff
@@ -99,5 +100,17 @@ func Delete(app string) error {
 		return errors.New(grpc.ErrorDesc(err))
 	}
 
+	taskList, err := tasks.Load(tasksTOMLFile)
+	if err != nil {
+		if err == tasks.ErrNoTaskFile {
+			debug(err.Error())
```

Comment:
We probably want to also return nil here, along with all the other task loading calls below. From what I'm reading here, when no task file exists, it'll show the error if `--debug` is displayed then attempt to call `.Run` on a `nil` object.